### ISSUE TITLE
Missing localization causing build warnings

### ISSUE THREAD

**Issue by abergie** [MicrosoftPremier/VstsExtensions#6]
```
2018-06-08T21:27:43.8406532Z ##[section]Starting: Check build quality
2018-06-08T21:27:43.8411239Z ==============================================================================
2018-06-08T21:27:43.8411329Z Task         : Build Quality Checks
2018-06-08T21:27:43.8411423Z Description  : Breaks a build based on quality metrics like number of warnings or code coverage.
2018-06-08T21:27:43.8411509Z Version      : 4.0.4
2018-06-08T21:27:43.8411581Z Author       : Microsoft Premier Services
2018-06-08T21:27:43.8411683Z Help         : [[Docs]](https://github.com/MicrosoftPremier/VstsExtensions/blob/master/BuildQualityChecks/en-US/overview.md)
2018-06-08T21:27:43.8411800Z ==============================================================================
2018-06-08T21:27:44.3100866Z SystemVssConnection exists true
2018-06-08T21:27:44.7390535Z Validating code coverage policy...
2018-06-08T21:27:45.1933530Z ##[warning]Resource file haven\'t set, can\'t find loc string for key: WaitForCodeCoverageData
2018-06-08T21:27:45.1934507Z WaitForCodeCoverageData
2018-06-08T21:27:47.7283040Z ##[warning]Resource file haven\'t set, can\'t find loc string for key: WaitForCodeCoverageData
2018-06-08T21:27:47.7284106Z WaitForCodeCoverageData
2018-06-08T21:27:49.9318110Z ##[warning]Resource file haven\'t set, can\'t find loc string for key: WaitForCodeCoverageData
2018-06-08T21:27:49.9319362Z WaitForCodeCoverageData
2018-06-08T21:27:52.3035306Z ##[warning]Resource file haven\'t set, can\'t find loc string for key: WaitForCodeCoverageData
2018-06-08T21:27:52.3036151Z WaitForCodeCoverageData
2018-06-08T21:27:54.6757581Z ##[warning]Resource file haven\'t set, can\'t find loc string for key: CoverageDataFound
2018-06-08T21:27:54.6758699Z CoverageDataFound
2018-06-08T21:27:54.6771221Z Total lines: 1881
2018-06-08T21:27:54.6771526Z Covered lines: 982
2018-06-08T21:27:54.6771728Z Code Coverage (%): 52.21
2018-06-08T21:27:54.8460689Z ##[warning]Resource file haven\'t set, can\'t find loc string for key: BaselineBuild
2018-06-08T21:27:54.8461307Z BaselineBuild 4536
2018-06-08T21:27:56.2217113Z ##[warning]Resource file haven\'t set, can\'t find loc string for key: CoverageDataFound
2018-06-08T21:27:56.2251215Z CoverageDataFound
2018-06-08T21:27:56.2256137Z Total lines: 1881
2018-06-08T21:27:56.2256319Z Covered lines: 982
2018-06-08T21:27:56.2256435Z Code Coverage (%): 52.21
2018-06-08T21:27:56.2281588Z ##[section]Finishing: Check build quality
```

Issue similar to: https://github.com/Microsoft/vsts-tasks/issues/5815

**Comment by almtcger** [comment]
Hi Aaron,

thanks for letting us know. The issue is caused by a version mismatch of the vsts-task-lib used in our task and one of our helper npm modules. I'll fix this with the next update (hopefully tomorrow).

René

**Comment by ReneSchumacher** [comment]
Hi Aaron,

the issue has been fixed in version 5.0.0 of the task, which will be released today. I'm closing the issue now. If there's anything else you need help with, please let us know.

René
